src/input/scroll.c

```diff
@@ -203,7 +203,16 @@ int scroll_decay(view_tile *position)
 
 int scroll_get_direction(const mouse *m)
 {
-    if (!m->is_inside_window && !m->is_touch) {
+    int is_inside_window = m->is_inside_window;
+    int width = screen_width();
+    int height = screen_height();
+    if (setting_fullscreen() && m->x < width && m->y < height)
+    {
```

Comment:
Please keep braces style in line with the other code (i.e. `{` on the same line as the `if`)